src/soc/sifive/fu540/src/init.S

```diff
@@ -1,4 +1,9 @@
-
+.globl __atomic_store_16
```

Comment:
Might be worth adding a comment here that it is a hack to satisfy the runtime?

Reply:
done.